app/forms/sufia/forms/batch_edit_form.rb

```diff
@@ -26,9 +26,9 @@ def initialize(model, current_ability, batch)
         def initialize_combined_fields(batch)
           combined_attributes = {}
           permissions = []
-          # For each of the files in the batch, set the attributes to be the concatination of all the attributes
+          # For each of the files in the batch, set the attributes to be the concatenation of all the attributes
           batch.each do |doc_id|
-            work = model_class.load_instance_from_solr(doc_id)
+            work = model_class.find(doc_id).to_solr
```

Comment:
I don't think you need the `to_solr` here, it's expecting an instance of `GenericWork` (or some kind of work)